src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingStonemason.java

```diff
@@ -96,7 +100,18 @@ public DOCraftingModule(final JobEntry jobEntry)
          */
         public @NotNull static OptionalPredicate<ItemStack> getStaticIngredientValidator()
         {
-            return CraftingUtils.getIngredientValidatorBasedOnTags(CRAFTING_STONEMASON);
+
+            OptionalPredicate<ItemStack> verifyStone = stack -> 
+            {
+                if(stack.is(Items.STONE) || stack.is(Items.NETHERRACK) 
+                || stack.is(Blocks.PRISMARINE.asItem()) || stack.is(Blocks.DARK_PRISMARINE.asItem()))
```

Comment:
This whitespacing irks me.

Also, perhaps worth introducing a new tag for this?

Reply:
I've thought about a new tag, and I may still do that, but that is a much bigger change, and I'm not sure we'll ever need to do this again.

Reply:
It'd be an extension point that lets modpacks add more stonemason-specific blocks if they have other things that cause weirdness.  (e.g. if they add even more stone-ish "storage blocks", or something else.)

And other than picking a name, it's very easy to add a new tag.  Just a few lines of code and a `runData`.

Reply:
I agree, a tag would be easy, and make extending things easier on the long term

Reply:
Ok. Switched to a tag